pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -91,6 +91,11 @@ func getDBImages(extensionState testhelpers.TestOptions, prevPromscaleVersion *s
 		return "timescale/timescaledb-ha:pg" + pgVersion + ".12-ts2.8.1-latest", dockerImageName, nil
 	}
 
+	// From Promscale 0.17.0 onwards the minimum extension version is 0.8.0
+	if prevPromscaleVersion != nil && prevPromscaleVersion.GE(semver.MustParse("0.17.0")) {
+		return "timescale/timescaledb-ha:pg" + pgVersion + ".12-ts2.9.1-latest", dockerImageName, nil
```

Comment:
Given image does not exist
```
❯ docker pull timescale/timescaledb-ha:pg12.12-ts2.9.1-latest
Error response from daemon: manifest for timescale/timescaledb-ha:pg12.12-ts2.9.1-latest not found: manifest unknown: manifest unknown
```
I think DB code team deprecated PG12. Maybe we can just use PG13 here?

Reply:
I had the minor version wrong, I'm on fire with mistakes in this release, the correct one is timescale/timescaledb-ha:pg12.13-ts2.9.1-latest. 

I was told it's not yet deprecated.